analyses/org.eclipse.tracecompass.incubator.xaf.ui/src/org/eclipse/tracecompass/incubator/internal/xaf/core/statemachine/variable/utils/CriticalPathState.java

```diff
@@ -13,8 +13,8 @@
 
 import java.util.Objects;
 
-import org.eclipse.tracecompass.analysis.graph.core.base.TmfEdge;
 import org.eclipse.tracecompass.analysis.os.linux.core.execution.graph.OsWorker;
+import org.eclipse.tracecompass.internal.analysis.graph.core.base.TmfEdge;
```

Comment:
If the objective is to remove all dependencies to the previous implementation then it should be analysis.graph.core.graph.historytree.TmfEdge and all the references to TmfEdge.EdgeType should be updated with OSEdgeContextState.

Reply:
TmfEdge.EdgeType was deprecated for a while and moved to internal as it was suggested in the patch that deprecated it.  The PR is to make XAF compile and not make it work. @MatthewKhouzam suggested to add a PR on top to make XAF work again before removing it from the code base.